kafkaex/kafka_ex
# Drop elixir < 1.5 support

* Switch to extra_applications to allow using kayrock in releases
* Add an overhead timeout in `server.ex` to allow timeout failures to happen where they should
* testing:
  * Use env vars in the docker-compose to ease the configuration.
  * Use several listeners for the Kafka brokers so that we don't have to detect the broker IPs
  * use random topic names to make sure each test is independent
